app.js

```diff
@@ -506,6 +512,12 @@ d.run(() => {
 				 * @todo Add description for the function and its params
 				 */
 				function(scope, cb) {
+					if (parseInt(appConfig.wsWorkers) < 1) {
```

Comment:
That's a wrong assumption. The number of workers should stay independent. I would prefer to re-use the config.peers.enabled for that purpose